<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListPatients</class>
 <widget class="QWidget" name="ListPatients">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="maximumSize">
      <size>
       <width>1080</width>
       <height>800</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="formTitle">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>List Patients</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QWidget" name="widget_2" native="true">
          <property name="styleSheet">
           <string notr="true">#widget_2 {
	background:#ffffff;
	background-image:none;
	border:1px solid #bbbbbb;
	border-radius:9px;
}

QWidget {
	background:#ffffff;
}

QTableWidget {
	border:1px solid #dddddd;
}

QPushButton {
	background:#fff;
	background-image:none;
	border-radius: 9px;
	border:1px solid #b1b1b1;
	padding:5px;
	padding-left:10px;
	padding-right:10px;
}

QPushButton:pressed {
	background-color: #555555;
	color: #ffffff;
}</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="leftMargin">
            <number>12</number>
           </property>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">#widget {
	border:1px solid #dddddd;
	border-radius: 9px;
}

QLineEdit {
	background:#f7f7f7;
	border:1px solid #dddddd;
	padding:2px;
}</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="leftMargin">
               <number>10</number>
              </property>
              <property name="topMargin">
               <number>10</number>
              </property>
              <property name="rightMargin">
               <number>10</number>
              </property>
              <property name="bottomMargin">
               <number>10</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Filters</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QGridLayout" name="gridLayout_2">
                  <property name="topMargin">
                   <number>10</number>
                  </property>
                  <property name="bottomMargin">
                   <number>10</number>
                  </property>
                  <item row="4" column="3">
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string>Children</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="5">
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>Telephone</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>Surname</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="3">
                   <widget class="QLabel" name="label_10">
                    <property name="text">
                     <string>Age</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>Nationality</string>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="5">
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>Inheritance</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="6">
                   <widget class="QLineEdit" name="filter_telephone"/>
                  </item>
                  <item row="1" column="6">
                   <widget class="QLineEdit" name="filter_fathersName"/>
                  </item>
                  <item row="1" column="5">
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>Father's Name</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="4">
                   <widget class="QLineEdit" name="filter_age"/>
                  </item>
                  <item row="3" column="2">
                   <widget class="QComboBox" name="filter_nationality">
                    <property name="styleSheet">
                     <string notr="true">QAbstractiItemView, QComboBox, QWidget {
	background:none;
}</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="4">
                   <widget class="QLineEdit" name="filter_name"/>
                  </item>
                  <item row="1" column="3">
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Name</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="2">
                   <widget class="QLineEdit" name="filter_surname"/>
                  </item>
                  <item row="4" column="4">
                   <widget class="QComboBox" name="filter_children">
                    <property name="styleSheet">
                     <string notr="true">QAbstractiItemView, QComboBox, QWidget {
	background:none;
}</string>
                    </property>
                    <item>
                     <property name="text">
                      <string/>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>0</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>1</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>2</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>3</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>4</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>5</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>6</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>7</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>8</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>9</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>10</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>11</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>12</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item row="4" column="6">
                   <widget class="QComboBox" name="filter_inheritance">
                    <property name="styleSheet">
                     <string notr="true">QAbstractiItemView, QComboBox, QWidget {
	background:none;
}</string>
                    </property>
                    <item>
                     <property name="text">
                      <string/>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>ΝΑΙ</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>ΟΧΙ</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item row="4" column="1">
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Cancer</string>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="2">
                   <widget class="QComboBox" name="filter_cancer">
                    <property name="styleSheet">
                     <string notr="true">QAbstractiItemView, QComboBox, QWidget {
	background:none;
}</string>
                    </property>
                    <item>
                     <property name="text">
                      <string/>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>ΝΑΙ</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>ΟΧΙ</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTreeWidget" name="treeWidget">
             <property name="styleSheet">
              <string notr="true">QScrollBar {
	background:none;
}</string>
             </property>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Export To Excel</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/icons/Excel 24x24</normaloff>:/icons/Excel 24x24</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>10</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Import From Excel</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/icons/Import 24x24</normaloff>:/icons/Import 24x24</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>10</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Print Patients List</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/icons/Print 32x32</normaloff>:/icons/Print 32x32</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>10</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Refresh List</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/icons/Refresh 24x24</normaloff>:/icons/Refresh 24x24</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
